# /!/usr/bin/python3
# *-*coding-utf8*-*

success={"user":"123456789","passwd":"1201201"}

# 异常场景
wrong_datas=[{"user":"234345","passwd":"hdh"},
    {"user":"111111134","passwd":"hdh"},
    {"user":"00000001","passwd":"hdh"}
             ]